mkdocstrings_handlers/vba/_handler.py

```diff
@@ -141,75 +150,79 @@ def collect(
     def render(
         self,
         data: VbaModuleInfo,
-        config: Mapping[str, Any],
+        config: MutableMapping[str, Any],
     ) -> str:
-        final_config = ChainMap(dict(copy.deepcopy(config)), self.default_config)
         template = self.env.get_template(f"module.html")
 
         # Heading level is a "state" variable, that will change at each step
         # of the rendering recursion. Therefore, it's easier to use it as a plain value
         # than as an item in a dictionary.
-        heading_level = final_config["heading_level"]
+        heading_level = config["heading_level"]
         try:
-            final_config["members_order"] = Order(final_config["members_order"])
+            config["members_order"] = Order(config["members_order"])
         except ValueError:
             choices = "', '".join(item.value for item in Order)
             raise PluginError(
-                f"Unknown members_order '{final_config['members_order']}', choose between '{choices}'."
+                f"Unknown members_order '{config['members_order']}', choose between '{choices}'."
             )
 
         return template.render(
-            config=final_config,
+            config=config,
             module=data,
             heading_level=heading_level,
             root=True,
         )
 
-    def update_env(self, md: Markdown, config: Dict[Any, Any]) -> None:
-        super().update_env(md, config)
+    def update_env(self, config: Dict[Any, Any]) -> None:
         self.env.trim_blocks = True
         self.env.lstrip_blocks = True
         self.env.keep_trailing_newline = False
         self.env.filters["crossref"] = do_crossref
         self.env.filters["multi_crossref"] = do_multi_crossref
         self.env.filters["order_members"] = do_order_members
 
-    def get_anchors(self, data: VbaModuleInfo) -> Tuple[str, ...]:
+    def get_aliases(self, identifier: str) -> Tuple[str, ...]:
+        """Get the aliases of the given identifier.
+
+        Aliases are used to register secondary URLs in mkdocs-autorefs,
+        to make sure cross-references work with any location of the same object.
+
+        Arguments:
+            identifier: The identifier to get aliases for.
+
+        Returns:
+            A tuple of aliases for the given identifier.
+        """
+        try:
+            data = self.collect(identifier, {})
+        except CollectionError:
+            return ()
         return data.path.as_posix(), *(p.signature.name for p in data.procedures)
```

Comment:
I have replicated previous behavior here, but I have to note that this seems incorrect to return both the path of the file, as well as all the procedure names within it. It's possible that the previous method (`get_anchors`) was confusing, and the "anchors" concept not well explained. The renaming to `get_aliases` is to make it more clear: it's not really about HTML anchors, it's about obtaining all the different "aliases" ("locations") for a given object, to correctly populate mkdocs-autorefs data. I know nothing about VBA, but surely `Procedure1` is not an alias of `Procedure2`? 

Aliases (and the identifier) are supposed to be strings that you can pass to `collect`.

Reply:
I will have to dig in and re-learn this code. A few years ago, I wrote the VBA handler by copying the Python handler and modifying it until it works. Even back then, I did not understand all the code. I'll try to go from this PR and see if I can build up a better understanding.